test/SAMtags/MM-orient.txt

```diff
@@ -0,0 +1,147 @@
+A	T
```

Comment:
please add comments as to what the columns are in this .dat file

Reply:
The .txt files are the expected output from parsing the .sam.  If I comment them it complicates comparisons.  They are documented in README.md.